lib/core_extensions.rb

```diff
@@ -124,7 +124,7 @@ def self.per_page
   def self.audited(*args)
     # do not audit data changes during db migrations
     super
-    self.auditing_enabled = false if Foreman.in_rake?('db:migrate')
+    self.disable_auditing if Foreman.in_rake?('db:migrate')
```

Comment:
What am I missing here https://github.com/collectiveidea/audited/blob/924cf56686c7eb07bda8e50864adcfbc1777a2dc/lib/audited/auditor.rb#L272? Even the Readme mentiones the auditing_enabled, not the disable_audigin https://github.com/collectiveidea/audited#disabling-auditing